### DISCUSSION TITLE
Curl_dedotdotify  An exception occurs

### DISCUSSION THREAD

**Discussion by zlz3132** [curl/curl#9188] [Q&A] [answered]
my curl version as follow:
curl 7.58.0 (x86_64-pc-linux-gnu) libcurl/7.58.0 OpenSSL/1.1.1 zlib/1.2.11 libidn2/2.0.4 libpsl/0.19.1 (+libidn2/2.0.4) nghttp2/1.30.0 librtmp/2.3
Release-Date: 2018-01-24
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtmp rtsp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz TLS-SRP HTTP2 UnixSockets HTTPS-proxy PSL 

Exception stack:
 Program terminated with signal SIGSEGV, Segmentation fault.
 #0  0xf72d9f98 in malloc () from /lib/libc.so.6
 [Current thread is 1 (Thread 0xf60ef450 (LWP 4391))]
 #0  0xf72d9f98 in malloc () from /lib/libc.so.6
 No symbol table info available.
 #1  0x00169198 in Curl_dedotdotify ()
 No symbol table info available.
 #2  0x001611a8 in seturl ()
 No symbol table info available.
 #3  0x0016206c in curl_url_set ()
 No symbol table info available.
 #4  0x0015f260 in Curl_connect ()
 No symbol table info available.
 #5  0x00157132 in multi_runsingle ()
 No symbol table info available.
 #6  0x00157b7e in curl_multi_perform ()
 No symbol table info available.
 #7  0x001505e6 in curl_easy_perform ()
 No symbol table info available.

**Comment by dfandrich** [comment, marked as answer]
Does it occur on ver. 7.84.0? That version is well over 4 years old and URL
handling was radically changed in 7.62.0.

**Reply by bagder** [reply]
and if it reproduces in a modern curl, this is a subject to file as an issue

**Comment by zlz3132** [comment]
tks，I will try it with the last version.
